_data/simple-icons.json

```diff
@@ -14910,6 +14910,11 @@
             "hex": "00A66C",
             "source": "https://stagetimer.io"
         },
+        {
+            "title": "Standard Notes",
+            "hex": "086DD6",
+            "source": "https://standardnotes.com"
```

Comment:
Are you using the logo in the header? Not matching up for me:

![CleanShot 2024-09-23 at 09 02 19](https://github.com/user-attachments/assets/7d0d148c-53ad-40c5-89f7-a19947a90bf2)